README.md

```diff
@@ -446,18 +446,23 @@ assets:
 
 ### Bootstrap
 
+***Gemfile***
 ```ruby
-gem "boostrap-sass" # 3.x
+gem "bootstrap-sass" # 3.x or:
 gem "bootstrap"     # 4.x
 ```
 
+***SCCS Inclusion***
```

Comment:
Why is this format change necessary? When it creates a deviation from my own standard documentation format, and provides no further context.

Reply:
When viewing the markdown compiled version there is absolutly no indication on the use of thoses code-blocks.

This was only a suggestions, and limited to the context of the modified bootstrap plugins but something should really be done on the whole documentation.

Reply:
What do you think codeblocks are for? They're literal examples.